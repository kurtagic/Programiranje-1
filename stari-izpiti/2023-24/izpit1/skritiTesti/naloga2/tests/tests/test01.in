1 10 27 384
9 25 444
9 7 439
7 26 490
9 26 530
9 3 500
9 13 44
3 13 190
5 17 211
9 22 1
5 10 577
3 3 399
5 10 733
3 8 374
3 16 49
0 9 676
9 23 161
5 13 535
1 5 248
6 5 669
5 13 732
9 16 221
5 13 412
7 17 138
7 24 105
5 13 562
1 17 375
3 5 141
3 7 737
7 5 57
6 13 371
9 17 549
9 17 249
1 5 447
5 3 474
4 16 407
6 22 195
9 14 36
5 15 419
5 25 462
7 26 549
6 5 210
3 17 513
5 7 249
1 13 732
4 25 384
4 25 702
1 14 143
0 14 662
5 25 677
9 7 502
6 25 13
2 25 355
3 16 694
2 3 296
0 17 3
1 9 314
6 24 521
3 17 89
3 23 278
4 17 200
0 13 550
3 15 235
4 13 55
5 4 206
4 26 238
0 17 519
4 2 708
5 26 365
7 16 292
5 2 603
7 9 406
5 8 731
1 15 311
1 2 187
7 15 186
5 26 45
1 17 612
9 15 206
4 26 533
8 25 566
9 20 575
2 8 663
6 6 246
9 15 488
9 25 372
5 14 499
1 2 688
0 13 307
3 14 304
2 5 594
8 11 522
6 23 212
6 3 17
5 4 250
4 16 22
1 11 733
8 6 520
2 6 342
8 5 546
9 14 211
2 3 146
4 16 241
2 5 206
3 13 30
7 1 136
7 19 626
5 17 388
1 6 636
2 6 394
3 12 623
9 11 22
2 22 384
3 23 609
7 10 519
0 16 281
9 2 458
6 1 630
2 9 184
1 2 657
5 0 175
6 21 138
4 14 295
3 20 14
8 3 130
3 6 578
7 25 643
5 20 338
4 17 441
4 2 300
6 23 394
5 17 628
1 16 444
1 6 593
1 16 25
6 22 73
5 9 258
4 17 503
7 20 694
1 26 216
5 13 716
6 0 305
0 14 707
9 18 722
7 7 8
6 11 337
9 23 60
5 7 556
8 7 666
0 5 264
2 13 734
4 4 332
1 2 571
3 20 686
0 18 310
5 14 206
7 13 9
0 5 43
4 6 457
3 17 414
0 6 98
5 17 471
3 16 625
5 20 343
6 6 605
5 24 405
2 14 271
5 24 568
5 17 460
2 15 677
9 12 655
8 16 42
9 15 518
8 19 597
4 24 362
5 20 384
6 22 224
7 4 614
8 22 650
6 22 223
4 5 716
0 1 213
4 23 421
4 25 48
1 22 568
5 19 663
5 17 382
1 1 111
5 19 138
6 19 701
2 17 717
7 24 53
4 26 36
5 10 19
9 13 364
9 8 253
8 3 417
8 2 474
2 6 479
8 23 159
0 14 477
9 26 138
9 12 596
1 14 1
4 24 341
7 4 56
3 11 181
8 6 736
4 8 700
1 2 17
9 25 339
7 2 215
2 25 483
4 0 593
0 3 235
8 8 356
0 9 33
5 13 305
6 7 547
0 6 549
5 2 79
2 5 518
7 19 209
0 1 192
7 8 621
8 13 508
1 20 650
5 21 497
3 19 64
6 26 698
2 8 657
0 1 175
7 26 515
4 4 14
5 20 199
4 16 222
3 16 503
4 17 714
9 10 599
8 25 205
4 15 139
1 10 520
0 14 45
4 13 692
6 9 394
4 15 371
2 6 95
8 14 129
2 8 704
8 21 393
6 18 425
8 9 296
1 23 723
5 10 439
0 23 738
1 1 95
5 11 316
9 9 406
6 22 518
9 12 357
5 22 152
0 7 587
8 14 516
5 25 627
3 3 34
3 16 555
7 25 330
4 15 650
8 7 358
1 0 402
8 24 137
1 8 114
6 4 24
3 5 239
1 18 280
9 10 628
8 13 472
4 2 669
4 23 311
3 16 39
8 0 362
5 14 533
1 8 637
2 14 650
6 16 78
3 26 689
3 13 401
1 25 336
5 17 76
2 2 559
3 6 675
7 5 38
6 21 525
9 8 224
4 13 609
9 26 241
2 9 558
1 6 327
5 0 54
7 14 209
2 19 364
0 8 186
3 14 426
0 13 610
8 12 5
7 10 190
8 10 415
9 2 112
7 13 737
0 20 204
3 12 436
5 15 445
0 19 626
9 6 369
7 3 131
6 19 142
2 21 42
8 9 541
5 6 168
7 2 447
5 14 240
3 23 663
1 16 582
0 14 56
7 1 225
8 7 114
7 3 518
3 8 229
0 26 604
5 21 337
5 5 238
9 5 74
2 16 156
8 17 158
5 16 374
6 9 112
5 1 538
9 8 63
8 3 424
2 2 118
7 13 1
9 2 219
8 24 491
5 23 456
3 20 499
0 6 311
9 24 513
4 0 82
8 4 671
6 14 209
5 2 653
6 2 492
1 12 188
8 21 706
2 6 716
8 26 106
3 26 683
5 12 676
7 1 168
8 16 424
0 8 636
1 18 312
4 19 211
4 9 208
8 24 684
2 2 335
9 11 115
1 13 412
8 6 77
4 14 632
4 20 736
0 18 679
7 1 409
5 16 172
2 9 414
5 17 159
2 26 293
2 11 495
7 14 58
9 0 229
3 9 87
5 12 580
4 26 263
0 23 613
3 8 643
