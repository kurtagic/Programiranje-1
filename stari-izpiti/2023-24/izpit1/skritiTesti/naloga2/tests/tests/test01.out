[593, 630, 708, 518, 671, 716, 736, 737, 731, 676, 733, 733, 676, 737, 707, 677, 694, 717, 722, 701, 736, 706, 650, 738, 684, 702, 698]
